internal/config/job.go

```diff
@@ -26,6 +27,7 @@ func (c *JobSubmit) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().Var(&c.TemplateVals, "set", "--set var=val")
 	cmd.Flags().BoolVar(&c.Attach, "attach", false, "waits until the job is completed, displaying the stdout and stderr streams")
 	cmd.Flags().DurationVar(&c.Timeout, "timeout", 0, "timeout when waiting for the job to be started, if 0 is specified, no timeout will be applied and the command will wait until completion or cancellation of the job (default 0)")
```

Comment:
In `sb apply`, we have `--wait` and `--wait-timeout`. Here, we have this `timeout` property that looks sort of related to `wait`. I'm curious whether this timeout is the equivalent of `--wait-timeout`?

Reply:
yes, it is.   I guess we could add an alias or something to make it more uniform